src/components/form-editor/form-editor.component.tsx

```diff
@@ -306,12 +328,15 @@ const FormEditor: React.FC = () => {
                     <a download={`${form?.name}.json`} href={window.URL.createObjectURL(downloadableSchema)}>
                       <Button
                         enterDelayMs={300}
-                        renderIcon={Download}
+                        renderIcon={isMaximized ? Minimize : Maximize}
                         kind={'ghost'}
-                        iconDescription={t('downloadSchema', 'Download schema')}
+                        iconDescription={
+                          isMaximized ? t('minimizeEditor', 'Minimize editor') : t('maximizeEditor', 'Maximize editor')
+                        }
                         hasIconOnly
                         size="md"
                         tooltipAlignment="start"
+                        onClick={handleToggleMaximize}
```

Comment:
What happens to the stuff related to downloading the sample schema?

Reply:
@denniskigen what do you mean by this? I am sorry I don't understand the question.

Reply:
This seems to already be on the main, I am not sure how the commit ended up in my commit.